content/blog/2020-05-11-subtitles/index.md

```diff
@@ -0,0 +1,126 @@
+
+#### The difficulty of collaborative captioning
+
+We're used to working together and have already collaborated on multiple projects. We even wrote [a post](/blog/2019/06/04/rromeo/) on this very blog about a package we submitted to rOpenSci software review.
+But this didn't help to find an efficient collaborative workflow for captioning. As mentioned earlier, one issue is that we had different captioning styles, and only realised it late in the project.
+
+Furthermore, even if Amara's subtitle editor is an amazing tool that allow you to easily pause, rewind, and advance the video while captioning, it does not support simultaneous editing of the subtitles. Comparatively from taking collaborative notes through EtherPad, this slows down the process quite a lot.
+
+Other communities, such as TED videos captioning community, disable entirely
+collaborative captioning. When someone starts working on a video, it disappears from the list of available tasks and they work on it alone.
+
+### About Amara
+
+#### Nice features
+
+Amara.org has a set of useful features that increase a bit your efficiency. The most notable ones in our opinion are:
+
+- keyboard shortcuts such as 'Tab' to play/pause the video, 'Shift+Tab' to go back in the video, 'Ctrl' to create a new subtitle, and 'Shift+Ctrl' to add a line break in a subtitle;
+- automatic deletion of leading whitespaces;
+- warnings regarding caption length, number of lines used, and caption reading speed (the longer the caption the longer it has to stay on screen so that viewers can read it!);
+- full versioning of the subtitles, we can still go back through all the subtitle versions we edited.
+
+#### Missing features
+
+But there are other features we could not find (they might exist but we missed them?):
+
+- an option to quickly merge subtitles. Very often, the subtitles are not split at the right place and you want to merge them to cut at the right time. A keyboard shortcut would be super useful here. It's even more annoying because when you try to select one subtitle to copy/paste it, amara removes your selection.
+- by default, subtitles start/stop exactly at the same time as the audio. Maybe it's not the recommended practice but we found it more comfortable to add some buffer (even a split second) to give smoother flow, and leave a bit more time for the reader.
+
+#### Proposed optimised workflow
+
+Given the amount of time required to achieve a perfect result (or even a "good enough" result as in this case), we are not sure it worth continuing this enterprise, at least as long as we don't find better tools / methods.
```

Comment:
I'm not sure it's really possible for us to answer this question. I think we would need to ask people who need the subtitles if (imperfect) auto-generated subtitles work for them or where exactly the threshold lies in their opinion.

Maybe it would be good to add a sentence mentioning this and suggesting that feedback on this specific point would be much appreciated.